src/ashpy/losses/executor.py

```diff
@@ -185,7 +199,7 @@ def __mul__(self, other: Union[Callable[..., float], float, int, tf.Tensor]):
             self._weight = lambda step: weight(step) * __other(step)
         return self
 
-    def __rmul__(self, other):
+    def __rmul__(self, other: Union[SumExecutor, Executor]):
         """See `__mul__` method."""
```

Comment:
```__mul__``` will actually be converted as _\_\_mul\_\__ . Here use one of the following:

```rst
``__mul__``
:code:`__mul__`
:meth:`ashpy.losses.Executor.__mul__() <ashpy.losses.executor.Executor.__mul__()>`
```
The last one is probably the best one.